python/lsst/meas/algorithms/readTextCatalogTask.py

```diff
@@ -47,6 +47,11 @@ class ReadTextCatalogConfig(pexConfig.Config):
         default=',',
         doc='Delimiter to use when reading text reference files.  Comma is default.'
     )
+    encoding = pexConfig.Field(
+        dtype=str,
+        default="utf-8",
+        doc="Encoding of text reference file."
```

Comment:
Do we really need to make this configurable? Can't we just make it `utf-8` forever and forget about it?